components/inputs/docs/input-date-time.md

```diff
@@ -194,15 +194,14 @@ The `<d2l-input-date-time>` component consists of a `<d2l-input-date>` and a `<d
 
 **Properties:**
 
-Note: All `*value` properties should be in ISO 8601 combined date and time format (`YYYY-MM-DDTHH:mm:ss.sssZ`) and in UTC time (i.e., do NOT localize to the user's timezone).
```

Comment:
Instead of removing this, can you change it to say something like "... `max-value`, `min-value` and `value` should be in ISO 8601..."